selinux/ceph.te

```diff
@@ -52,7 +53,7 @@ files_pid_file(ceph_var_run_t)
 allow ceph_t self:process { signal_perms };
 allow ceph_t self:fifo_file rw_fifo_file_perms;
 allow ceph_t self:unix_stream_socket create_stream_socket_perms;
-allow ceph_t self:capability { setuid setgid dac_override };
+allow ceph_t self:capability { setuid setgid dac_override dac_read_search };
```

Comment:
We have been doing this for quite some time. However, there was a change in kernel [1] and now, the kernel generates the `dac_read_search` even though it eventually grants the permission to read the file because of `dac_override` that we have already had in our policy for a while.

[1] https://danwalsh.livejournal.com/77140.html

Reply:
i've read the very same blog.